app_integrations/apps/__init__.py

```diff
@@ -2,6 +2,45 @@
 import importlib
 import os
 
+from app_integrations.exceptions import AppException
+
+
+class StreamAlertApp(object):
+    """Class to be used as a decorator to register all AppIntegration subclasses"""
+    _apps = {}
+
+    def __new__(cls, app):
+        StreamAlertApp._apps[app.type()] = app
+        return app
+
+    @classmethod
+    def get_app(cls, app_type):
+        """Return the proper app integration class for this service
+
+        Args:
+            config (AppConfig): Loaded configuration with service, etc
+            init (bool): Whether or not this class should be instantiated with
+                the config that has been passed in
+
+        Returns:
+            AppIntegration: Subclass of AppIntegration corresponding to the config
```

Comment:
Can you also add a `Raises` section to the docstring, which we use when explicitly raising an exception

Reply:
@austinbyers done! thank you :)